keyboards/boardsource/5x12/config.h

```diff
@@ -4,7 +4,7 @@
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xF7E0
```

Comment:
This should also be changed for `boardsource/4x12/config.h` - that was my intention with suggesting it on https://github.com/qmk/qmk_firmware/pull/10043#pullrequestreview-468960656

Your PR to the VIA repo will need to be updated as well.

```suggestion
#define VENDOR_ID       0x4273 // "Bs" - Boardsource
```

Reply:
update here for 5x12, but the 4x12 PR has already been merged. Opened new PR to fix it https://github.com/qmk/qmk_firmware/pull/10306